utils/s3/aws_errors.cc

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#if __has_include(<rapidxml.h>)
+#include <rapidxml.h>
+#else
+#include <rapidxml/rapidxml.hpp>
+#endif
+
+#include "utils/s3/aws_errors.hh"
+#include <unordered_map>
+
+namespace aws {
+static const std::unordered_map<std::string, const aws_error> aws_error_map{
```

Comment:
probably `std::string_view` is good enough here?

Reply:
sure, done